test/error.js

```diff
@@ -1,19 +1,26 @@
 'use strict';
 
-const test = require('tap').test;
-
+const expect = require('expect');
 const gulp = require('gulp');
-const task = require('../');
+const { pipe } = require('mississippi');
 const path = require('path');
+const PluginError = require('plugin-error');
+const task = require('../');
 
 const filename = path.join(__dirname, './fixtures/pug-error.pug');
 
-test('should emit errors of pug correctly', function(t) {
-  gulp.src(filename)
-    .pipe(task()
-      .on('error', function(err) {
-        t.ok(err);
-        t.ok(err instanceof Error);
-        t.end();
-      }));
+describe('error', function() {
+  it('should emit errors of pug correctly', function(done) {
+    pipe(
+      [gulp.src(filename), task()],
+      (err) => {
+        try {
```

Comment:
Thrown errors here don't bubble up to the test?

Reply:
This is related to not using `miss.concat` in this test: I was just checking for the existence of an error and did not care about the content.

Reply:
I have to keep this style to pass the error to `done`.